SWARAM-UI-Documentation/components.css

```diff
@@ -0,0 +1,503 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+:root{
+    --primary:#aa3a3a;
+    --secondary:#C4C6A6;
+    --padding-1:0.5rem;
+    --padding-2:0.8rem;
+    --padding-3:0.4rem;
+    --font-size-text:1.1rem;
+    --font-size-small:0.8rem;
+    --font-size-title:01.4rem;
+    --font-family:Roboto;
+    --margin-right:0.6rem;
+}
+
+body{
+    margin:0;
+}
+
+.bg-primary{
+    background-color: var(--primary);
+    color:white;
+}
+
+.text-color-primary{
+    color:var(--primary)
+}
+
+.flex-horizontal{
+    display: flex;
+}
+
+/** Avatar **/
+
+.avatar-container{
+    position: relative;
+}
+
+.img-size-medium{
```

Comment:
You can use class-name as 'md' for medium , 'lg' for large and 'sm' for small

Reply:
I didn't want to name it in short forms. I get confused sometimes